pre_commit/commands/run.py

```diff
@@ -152,7 +152,9 @@ def _run_single_hook(classifier, hook, args, skips, cols):
 
         # Print a message if failing due to file modifications
         if file_modifications:
-            output.write('Files were modified by this hook.')
+            output.write('Files were modified by this hook.' \
+                         'Please review them in Git\'s staging area, ' \
+                         'and repeat the commit.')
```

Comment:
seems fine to add more context here

maybe something like this?

```python
output.write(
    'Files were modified by this hook.  '
    'Review the changes, `git add`, and repeat the commit.',
)
```

hmmm, but it's not terribly uncommon for this message to show multiple times with lots of fixers (it's a bit redundant in that case) and now it'll wrap on narrow terminals 🤔 

(the change won't be in the staging area, but will need to be added there)